### PR TITLE
 add(epp): hp storeonce refactoring

### PR DESCRIPTION
**Pull request by garnier-quentin**
## Target version

- [ ] 20.10.x (staging)
- [ ] 21.04.x (staging)
- [ ] 21.10.x (staging)
- [ ] 22.04.x (staging)
- [ ] Cloud (staging)
- [X] Plugin Packs (staging)
- [ ] 22.10.x (next)

### CODE REVIEW

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-storage-hp-storeonce3-restapi.md [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+| *serviceset_alias*#serviceset.space.usage.bytes         | Space used on the service set          | B     |
+| *serviceset_alias*#serviceset.deduplication.ratio.count | Deduplication ratio on the service set |       |
+
+</TabItem>
+</Tabs>
+
+## Prérequis
+
+Ce Pack supporte uniquement les versions 3.x des équipements HP StoreOnce.
+Pour la supervision, un utilisateur avec les droits en lecture est nécessaire. Référez vous à la documentation officielle: https://support.hpe.com/hpesc/public/docDisplay?docId=c05273975
```
Comment (lchrdn) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-storage-hp-storeonce3-restapi.md:68]:
```suggestion
Pour la supervision, un utilisateur avec les droits en lecture est nécessaire. Référez-vous à la documentation officielle: https://support.hpe.com/hpesc/public/docDisplay?docId=c05273975
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-storage-hp-storeonce4-restapi.md [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+| *store_name*#store.user.space.usage.bytes    | User space used on the catalyst store     | B     |
+| *store_name*#store.deduplication.ratio.count | Deduplication ratio on the catalyst store |       |
+
+</TabItem>
+</Tabs>
+
+## Prérequis
+
+Ce Pack supporte uniquement les versions 4.x des équipements HP StoreOnce.
+Pour la supervision, un utilisateur avec les droits en lecture est nécessaire. Référez vous à la documentation officielle: https://hewlettpackard.github.io/storeonce-rest/index.html
```
Comment (lchrdn) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-storage-hp-storeonce4-restapi.md:76]:
```suggestion
Pour la supervision, un utilisateur avec les droits en lecture est nécessaire. Référez-vous à la documentation officielle : https://hewlettpackard.github.io/storeonce-rest/index.html
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-storage-hp-storeonce3-restapi.md [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+```
+
+La commande devrait retourner un message de sortie similaire à :
+
+```bash
+OK: Cluster 'TABZ37161Q01' status: ok, Usage Total: 62.17 TB Used: 59.78 TB (96.15%) Free: 2.39 TB (3.85%), Dedup Ratio: 14.51 | 'TABZ37161Q01#cluster.space.usage.bytes'=65731753164800B;;;0;68360696004608 'TABZ37161Q01#cluster.deduplication.ratio.count'=14.51;;;0;
+```
+
+La liste de toutes les options complémentaires et leur signification peut être
+affichée en ajoutant le paramètre `--help` à la commande:
```
Comment (lchrdn) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-storage-hp-storeonce3-restapi.md:144]:
```suggestion
affichée en ajoutant le paramètre `--help` à la commande :
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-storage-hp-storeonce3-restapi.md [hunk 4]
```diff
@@ -0,0 +1,165 @@
...
+
+```bash
+$CENTREONPLUGINS$/centreon_hp_storeonce3_restapi.pl \
+    --plugin=storage::hp::storeonce::3::restapi::plugin \
+    --mode=cluster-usage \
+    --help
+```
+
+Tous les modes disponibles peuvent être affichés en ajoutant le paramètre 
+`--list-mode` à la commande:
```
Comment (lchrdn) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-storage-hp-storeonce3-restapi.md:154]:
```suggestion
`--list-mode` à la commande :
```

File: pp/sidebarsPp.js [hunk 5]
```diff
@@ -2160,7 +2160,11 @@ module.exports = {
         },
         {
           type: 'doc',
-          id: 'integrations/plugin-packs/procedures/hardware-storage-hp-storeonce-restapi'
```
Comment (lchrdn) [inline pp/sidebarsPp.js:2163]: if you delete hardware-storage-hp-storeonce-restapi from sidebarspp.js, the old deprecated page with redirection to the new ones won't be accessible?

Comment (garnier-quentin) [reply pp/sidebarsPp.js:2163]: it will be accessible from the pack manager. we don't want to be accessible from the menu.

**Review by lchrdn — APPROVED** [review]

**Pull request merged**